sphinx/builders/_epub_base.py

```diff
@@ -27,8 +27,9 @@
 
 try:
     from PIL import Image
+    IMAGE = True
 except ImportError:
-    Image = None
+    IMAGE = False
```

Comment:
If we need an explicit flag, I'd rather name this `PILLOW_AVAILABLE`.

Reply:
works for me